Makefile

```diff
@@ -20,7 +20,6 @@ proto:
 		echo "error: protoc not installed" >&2; \
 		exit 1; \
 	fi
-	go get -u -v github.com/golang/protobuf/protoc-gen-go golang.org/x/tools/cmd/stringer
```

Comment:
Should we keep this?

Though this works on travis, someone trying to run `make proto` locally may fail because of these two missing packages...

Reply:
I'm fine with reverting this, but then we should change `make proto` in `vet.sh` to run `go generate`.  Otherwise you end up with the latency of `go get` every time you run `vet.sh`, which is not really desirable.

Alternatively, we can check `$GOPATH/.....` for these two things and fail with a friendlier error if they are not found.

Thoughts?